WebDriverAgentLib/Commands/FBDebugCommands.m

```diff
@@ -43,4 +46,25 @@ + (NSArray *)routes
   return FBResponseWithStatus(FBCommandStatusNoError, @{ @"tree": (accessibleTreeType ? application.fb_accessibilityTree : application.fb_tree) ?: @{} } );
 }
 
++ (id<FBResponsePayload>)handleGetSourceCommand:(FBRouteRequest *)request
+{
+  FBApplication *application = request.session.application ?: [FBApplication fb_activeApplication];
+  if (!application) {
```

Comment:
Or instead of adding asserts for that everywhere we could handle that in FBapplication